includes/Reply.hpp

```diff
@@ -2,6 +2,8 @@
 #ifndef REPLY_HPP
 # define REPLY_HPP
 
+# define user(nick, username, userIP)(":" + nick + "!" + username + "@" + userIP)
```

Comment:
Should we write the macro "user" also on caps? -> USER

Reply:
yup. will change it now